src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -0,0 +1,100 @@
+import { Serif, space } from "@artsy/palette"
+import React from "react"
+import { media } from "styled-bootstrap-grid"
+import { BorderBox, Box } from "Styleguide/Elements/Box"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { Image } from "Styleguide/Elements/Image"
+import { Separator } from "Styleguide/Elements/Separator"
+import { Spacer } from "Styleguide/Elements/Spacer"
+import styled from "../../../../node_modules/styled-components"
+
+const SummaryContainer = styled(Flex)`
+  margin: ${space(2)}px;
+  ${media.sm`
+  margin: ${space(3)}px;
```

Comment:
Hmm there's a formatting issue here. I guess prettier doesn't respect the `media.*` pattern?